src/Components/GiveUpModal.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+class GiveUpModal extends React.Component {
```

Comment:
For components like this one, you can look into creating functional components rather than class components. 

For more information about the difference check this article out. They go into detail about when might be the right time to use one over the other. https://medium.com/@Zwenza/functional-vs-class-components-in-react-231e3fbd7108